src/components/content/catalog/services/update/UpdateService.tsx

```diff
@@ -267,7 +276,31 @@ function UpdateService({
                             </Row>
                         </Upload>
                     </div>
-                    <div>{oclDisplayData.current}</div>
+                    <div>
+                        {files.current.length > 0 ? (
+                            <div className={registerStyles.registerButtons}>
+                                <Row>
+                                    <Form.Item
+                                        name='isRemoveServiceTemplateUntilApproved'
+                                        label={
+                                            <span style={{ fontWeight: 'bold' }}>
+                                                RemoveServiceTemplateUntilApproved
+                                            </span>
+                                        }
+                                    >
+                                        <Space wrap>
+                                            <Checkbox
```

Comment:
checkbox must be disabled in the modal after the update is submitted and enabled again only when submit fails.